activitypub.php

```diff
@@ -161,8 +126,6 @@ function plugin_settings_link( $actions ) {
 	)
 );
 
-// Load integrations.
-require_once __DIR__ . '/integration/load.php';
```

Comment:
Can we delete this? It is not the included folder!?!

Reply:
It's moved to [line 34](https://github.com/Automattic/wordpress-activitypub/pull/1051/files#diff-d33bfd446d4ad05f88e9c8e907f31b2e4c06eca40c7f28cee7e58f3a2302958bR34), to group with the other includes.